testing/test_mercurial.py

```diff
@@ -114,7 +114,7 @@ def test_parse_no_worktree(tmpdir):
 def version_1_0(wd):
     wd('hg branch default')
     wd.commit_testfile()
-    wd('hg tag 1.0 -u test -d "0 0"')
+    wd('hg tag 1.0.0 -u test -d "0 0"')
```

Comment:
What's happening here? Do mercurial tags now require full semver and drop support for N.N numbers?